mxcube3/routes/qutils.py

```diff
@@ -157,14 +283,18 @@ def _handle_dc(sample_id, node):
     parameters.pop('acquisitions')
     parameters.pop('acq_parameters')
     parameters.pop('centred_position')
- 
+    queueID = node._node_id
+    enabled, state = get_node_state(queueID)
     res = {"label": "Data Collection",
            "type": "DataCollection",
            "parameters": parameters,
            "checked": node.is_enabled(),
```

Comment:
Be a bit careful here you seem to be assigning "checked" twice here and on line 295